tests/config/pluggable_state.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2022 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: dapr.io/v1alpha1
+kind: PluggableComponent
+metadata:
+  name: redis-pluggable
+spec:
+  type: state
+  version: v1
+---
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: pluggable-statestore
+spec:
+  type: state.redis-pluggable
+  version: v1
+  initTimeout: 20s
+  metadata:
+    - name: redisHost
+      secretKeyRef:
+        name: redissecret
+        key: host
+    - name: redisPassword
+      value: ""
+scopes:
+  - stateapp-pluggable
+---
+# TODO: Remove once PluggableComponents feature is finalized
```

Comment:
You can turn on this feature by test app.

Reply:
can you send an example of this?

I see others doing the same like: [resiliency](https://github.com/dapr/dapr/blob/master/tests/config/app_resiliency.yaml#L1) and [app healthcheck](https://github.com/dapr/dapr/blob/master/tests/config/app_healthcheck.yaml#L15)

Reply:
It is just in the same file here then. It is OK. I will not the PR for this.